src/lib/es5.d.ts

```diff
@@ -1463,7 +1463,9 @@ type Extract<T, U> = T extends U ? T : never;
 /**
  * Construct a type with the properties of T except for those in type K.
  */
-type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
+type Omit<T, K extends keyof any, Keys extends keyof T = Exclude<keyof T, K>> = {
+    [P in Keys]: T[P];
+};

```

Comment:
This can be implemented using mapped types now:
```suggestion
type Omit<T, K extends keyof any> = {
    [P in keyof T as P extends K ? never : P]: T[P];
};
```